frontend/src/lib/UriUtil.ts

```diff
@@ -100,21 +100,17 @@ function isHttps(): boolean {
 /**
  * Run SVG strings through DOMPurify to prevent Javascript execution
  */
-function sanitizeSvg(uri: string): string {
+export function sanitizeSvg(uri: string): string {
```

Comment:
It looks like you're not the originator of this function, but it feels like it could use a much better name. "sanitizeSVG" sounds like it could be doing all sorts of things (maybe it just strips extra whitespace or something similarly anodyne!) - but if I understand, the real impact is that it removes potential XSS vulnerabilities.

Could we rename this to something that makes this a bit clearer - like `xssSanitizeSvg`?

Reply:
Hmm. To me, `sanitize` pretty directly points at this behavior, but I agree it is kinda ambiguous/unclear. Ideally it would be clearer by returning a `SafeSvg` type or something, but short of that `xssSanitizeSvg` does seem better.